client/state/signup/preview/selectors.js

```diff
@@ -16,3 +16,17 @@ import 'state/signup/init';
  */
 export const isSitePreviewVisible = ( state ) =>
 	get( state, [ 'signup', 'preview', 'isVisible' ], false );
+
+/**
+ * Returns true if a plans step exists and is skipped in the signup flow
+ *
+ * @param   {object}  state The current client state
+ * @returns  {boolean} denoting whether the plans step existed AND it was skipped
+ */
+export const isPlanStepExistsAndSkipped = ( state ) => {
```

Comment:
Why is this selector under signup/preview/selectors.js file? Shouldn't be in a different folder since isPlanStepExistsAndSkipped Is not related to site previews?